### PR TITLE
importccl: update parquet vendor

### PR DESCRIPTION
**Pull request by msbutler**
importccl: update parquet vendor to v0.10.0
    
Release note: None

Release justification: bug fixes and low-risk updates to new functionality

### CODE REVIEW

File: DEPS.bzl [hunk 1]
```diff
@@ -2448,10 +2458,10 @@ def go_deps():
         name = "com_github_fraugster_parquet_go",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/fraugster/parquet-go",
-        sha256 = "66beb8f2218c31ca4ca3d3dbcc91c256f3379750ade924016f4179982446edd7",
-        strip_prefix = "github.com/fraugster/parquet-go@v0.6.1",
+        sha256 = "68aa69311763a3c816552da6b2571272f996bc111237cecbb171dd7d41bb9346",
+        strip_prefix = "github.com/fraugster/parquet-go@v0.8.0",
```
Comment (stevendanna) [inline DEPS.bzl:2462]: Looks like the latest is 0.9.0, want to bump so close to their most recent release, but figured I'd point it out.

**Review by stevendanna — APPROVED** [review]
LGTM.

Sometimes we include the highlights of or links to the changelog for the changes in the commit message.

File: DEPS.bzl (hunk 1 above)
Comment (msbutler) [reply DEPS.bzl:2462]: yeah, there's a known regression in 0.9.0 they're working on :/

Hope to update to latest release during stability.

**Comment by msbutler** [comment]
TFTR!

bors r=stevendanna

**Comment by msbutler** [comment]
bors r=stevendanna

**Pull request merged**